specification.md

```diff
@@ -1148,7 +1284,7 @@ Our defined function definition can then we referenced in a workflow [action](#A
 }
 ```
 
-#### Using Functions for RPC Service Invocations
+#### Using Functions for gRPC Service Invocations
```

Comment:
This breaks the two preceding `[…](#using-functions-for-rpc-service-invocations)` links.